server/lib/url.js

```diff
@@ -29,19 +30,21 @@ module.exports.external = uuid => {
 };
 
 module.exports.stream = (metadatum, options) => {
-	const streamUrl = `http://www.ft.com/stream/${metadatum.taxonomy}Id/${metadatum.idV1}`;
+	const fetch = wrapFetch(nodeFetch, {
```

Comment:
This will create a new instrumented `fetch` every times `stream` is run, which isn't particularly expensive but I'm not sure it's going to be garbage-collectable